lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.spec.ts

```diff
@@ -233,6 +233,38 @@ describe('CardViewTextItemComponent', () => {
             renderChipsForMultiValuedProperties(cardViewTextItemObject, true, 3, '1.1', '2.2', '3.3');
         });
 
+        it('should only render new chip when provided value is valid for specified validators set', async () => {
+            const cardViewTextItemObject = {
+                label: 'Text label',
+                value: [1.1, 2.2, 3.3],
+                key: 'textkey',
+                editable: true,
+                multivalued: true,
+                type: 'float'
+            };
+            component.editable = true;
+            renderChipsForMultiValuedProperties(cardViewTextItemObject, true, 3, '1.1', '2.2', '3.3');
+            const floatValidator: CardViewItemValidator = new CardViewItemFloatValidator();
+            component.property.validators = [floatValidator];
+            const inputElement = fixture.debugElement.query(By.css('[data-automation-id="card-textitem-editchipinput-textkey"]')).nativeElement;
+            component.addValueToList({ value: 'abcd', chipInput: inputElement, input: inputElement });
```

Comment:
input seems deprecated, can we use different approach?
<img width="1159" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/020d0da8-b0e2-4105-8c58-315f8824749f">

Reply:
Sure, fixed to do this other way